mycli/main.py

```diff
@@ -207,13 +217,16 @@ def read_my_cnf_files(self, files, keys):
                 pass
 
         sections = ['client']
+        if self.login_path and self.login_path != 'client':
+            sections.append(self.login_path)
+
         if self.defaults_suffix:
-            sections.append('client{0}'.format(self.defaults_suffix))
+            sections.extend([sect + self.defaults_suffix for sect in sections])
```

Comment:
Can a user specify both `login_path` and `defaults_sufix` in the command line? If they do, are we supposed to append the `defaults_suffix` to the `login_path`? That doesn't sound right to me. But I've never used login_path before so I'm merely speaking from my gut feeling.